src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -210,6 +211,10 @@ export default {
 		},
 	},
 
+	mounted() {
+		this.loadPhotoUrl()
```

Comment:
will this catch a contact change if the component stays mounted?

Reply:
Good point. Not yet. I suppose this needs a watcher